src/style.css

```diff
@@ -1 +1,148 @@
-/* add styles here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  margin: 0;
+}
+
+.stars ul {
+  list-style: none;
+  padding: 0;
+  display: flex;
+}
```

Comment:
Since `.stars ul` is already using a class for styling, the tag name `ul` is not necessary and can be replaced with a class like `.stars__list` for consistency and to follow best practices.